go/vt/vttablet/onlineddl/vrepl.go

```diff
@@ -208,7 +208,7 @@ func (v *VRepl) executeAnalyzeTable(ctx context.Context, conn *dbconnpool.DBConn
 		defer conn.ExecuteFetch(sqlDisableFastAnalyzeTable, 1, false)
 	}
 
-	parsed := sqlparser.BuildParsedQuery(sqlAnalyzeTable, tableName)
+	parsed := sqlparser.BuildParsedQuery(sqlAnalyzeTableLocal, tableName)
```

Comment:
OK, so this is only called as part of `analyze` step where it makes sense not to replicate it. Versus the cutover step where we continue to use the replicated variant.

Reply:
Correct!